src/material/dialog/dialog.spec.ts

```diff
@@ -28,7 +28,11 @@ import {MatDialogContainer} from './dialog-container';
 import {OverlayContainer, ScrollStrategy, Overlay} from '@angular/cdk/overlay';
 import {ScrollDispatcher} from '@angular/cdk/scrolling';
 import {A, ESCAPE} from '@angular/cdk/keycodes';
-import {dispatchKeyboardEvent, createKeyboardEvent} from '@angular/cdk/testing/private';
+import {
+    dispatchKeyboardEvent,
```

Comment:
It seems like there's too much indentation here. We use two spaces for indentation.

Reply:
done